Airflow retirement public doc updates

SunilVeldurthi opened the pull request.

SunilVeldurthi committed:
Airflow retirement public doc updates

kromerm reviewed (approved):
#sign-off

ktoliver commented on articles/data-factory/airflow-configurations.md:
```diff
@@ -10,8 +10,10 @@ ms.date: 02/13/2025
-> [!NOTE]
-> This feature is in public preview. Workflow Orchestration Manager is powered by Apache Airflow.
+> [!IMPORTANT]
+> Workflow Orchestration Manager (powered by Apache Airflow) will be permanently retired in Azure Data Factory (ADF) on December 31, 2025.  This Feature is now available in Microsoft Fabric. [Learn more](https://learn.microsoft.com/en-us/fabric/data-factory/apache-airflow-jobs-concepts?source=docs) 
```
```suggestion
> Workflow Orchestration Manager (powered by Apache Airflow) will be permanently retired in Azure Data Factory on December 31, 2025. The feature is now available in Microsoft Fabric. [Learn more](https://learn.microsoft.com/fabric/data-factory/apache-airflow-jobs-concepts?source=docs)
>
```

ktoliver commented on articles/data-factory/airflow-configurations.md:
```diff
@@ -10,8 +10,10 @@ ms.date: 02/13/2025
-> This feature is in public preview. Workflow Orchestration Manager is powered by Apache Airflow.
+> [!IMPORTANT]
+> Workflow Orchestration Manager (powered by Apache Airflow) will be permanently retired in Azure Data Factory (ADF) on December 31, 2025.  This Feature is now available in Microsoft Fabric. [Learn more](https://learn.microsoft.com/en-us/fabric/data-factory/apache-airflow-jobs-concepts?source=docs) 
+> We recommend migrating all Workflow Orchestration Manager (Apache Airflow in ADF) workloads to Data Workflows (Apache Airflow in Microsoft Fabric) to take advantage of the expanded capabilities before December 31, 2025.
```
```suggestion
> We recommend that you migrate all Workflow Orchestration Manager (Apache Airflow in Azure Data Factory) workloads to Data Workflows (Apache Airflow in Microsoft Fabric) to benefit from expanded capabilities before December 31, 2025.
```

ktoliver commented on articles/data-factory/airflow-configurations.md:
```diff
@@ -10,8 +10,10 @@ ms.date: 02/13/2025
+> [!IMPORTANT]
+> Workflow Orchestration Manager (powered by Apache Airflow) will be permanently retired in Azure Data Factory (ADF) on December 31, 2025.  This Feature is now available in Microsoft Fabric. [Learn more](https://learn.microsoft.com/en-us/fabric/data-factory/apache-airflow-jobs-concepts?source=docs) 
+> We recommend migrating all Workflow Orchestration Manager (Apache Airflow in ADF) workloads to Data Workflows (Apache Airflow in Microsoft Fabric) to take advantage of the expanded capabilities before December 31, 2025.
+>For additional guidance or support during your migration to Fabric Airflow, please contact Microsoft Support team.
```
```suggestion
>
> For more information or for support during your migration to Apache Airflow in Microsoft Fabric, contact Microsoft Support.
```

ktoliver reviewed (commented):
Edits for Microsoft style

ktoliver committed:
Update articles/data-factory/airflow-configurations.md

ktoliver committed:
Update articles/data-factory/airflow-configurations.md

ktoliver committed:
Update articles/data-factory/airflow-configurations.md

ktoliver committed:
Update airflow-configurations.md

ktoliver committed:
Update airflow-get-ip-airflow-cluster.md

ktoliver committed:
Update airflow-import-dags-blob-storage.md

ktoliver committed:
Update airflow-install-private-package.md

ktoliver committed:
Update airflow-pricing.md

ktoliver committed:
Update airflow-sync-github-repository.md

ktoliver committed:
Revise retirement notice for Workflow Orchestration Manager

Updated migration guidance for Workflow Orchestration Manager to Microsoft Fabric.

ktoliver committed:
Update concepts-workflow-orchestration-manager.md

ktoliver committed:
Update create-airflow-environment.md

ktoliver committed:
Revise retirement notice and migration guidance

Updated the retirement notice for Workflow Orchestration Manager and added information about migration to Microsoft Fabric.

ktoliver committed:
Revise retirement notice and migration instructions

Updated the retirement notice and migration guidance for Workflow Orchestration Manager.

ktoliver committed:
Update enable-azure-key-vault.md

ktoliver committed:
Revise migration instructions for Workflow Orchestration Manager

Updated the migration guidance for Workflow Orchestration Manager to clarify the transition to Microsoft Fabric and improve support information.

ktoliver committed:
Update password-change-airflow.md

ktoliver committed:
Update rest-apis-for-airflow-integrated-runtime.md

ktoliver committed:
Revise tutorial for running pipelines with Airflow

Updated the tutorial to clarify the migration from Workflow Orchestration Manager in Azure Data Factory to Microsoft Fabric and improved the introductory paragraph.
